test/e2e/simple_ovs_bridge.go

```diff
@@ -0,0 +1,50 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("NodeNetworkState", func() {
+	var ()
+	Context("when desiredState is configured", func() {
+		Context("with an ovs bridge up with no ports", func() {
+			BeforeEach(func() {
+				updateDesiredState(ovsBrUpNoPorts(bridge1))
```

Comment:
We should specify desired config only via NodeNetworkConfigurationPolicy. Setting those via NodeNetworkState will be likely deprecated (it was never supported for users, but we may drop it even from our internal implementation). See https://github.com/nmstate/kubernetes-nmstate/blob/master/test/e2e/default_bridged_network_test.go#L68

Reply:
I would suggest leave this like linux-bridge and do a follow up PR remove NodeNetworkState desiredState use all together from e2e tests.